src/PrismicLink.svelte

```diff
@@ -41,6 +42,7 @@
 	});
 
 	$: resolvedRel = typeof rel === "string" ? rel : linkAttrs.rel;
+	$: resolvedText = field?.text;
```

Comment:
💡 #idea: I think we can use `field?.text` directly rather than within a reactive statement. Could you try it out? I'm not 100% sure.

Reply:
Oh, great! Really better and working, thanks 🙏